railgun/README.md

```diff
@@ -1,46 +1,126 @@
+[![Build Status](https://github.com/kong/kubernetes-ingress-controller/workflows/Test/badge.svg)](https://github.com/kong/kubernetes-ingress-controller/actions?query=branch%3Amaster+event%3Apush)
+[![codecov](https://codecov.io/gh/Kong/kubernetes-ingress-controller/branch/main/graph/badge.svg?token=S1aqcXiGEo)](https://codecov.io/gh/Kong/kubernetes-ingress-controller)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/Kong/kong/blob/master/LICENSE)
+[![Twitter](https://img.shields.io/twitter/follow/thekonginc.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=thekonginc)
 
-## Usage
+# Kong for Kubernetes
 
-To run the dev environment currently, you need a Kubernetes cluster (such as [kind](https://github.com/kubernetes-sigs/kind)):
+Use [Kong][kong] for Kubernetes [Ingress][ingress].
+Configure [plugins][kong-hub], health checking,
+load balancing and more in Kong
+for Kubernetes Services, all using
+Custom Resource Definitions(CRDs) and Kubernetes-native tooling.
 
-```shell
-$ kind create cluster --name kong-test
-kubectl cluster-info --context kind-kong-test
-```
+[**Features**](#features) | [**Get started**](#get-started) | [**Documentation**](#documentation) | [**Master branch builds**](#master-branch-builds) | [**Seeking help**](#seeking-help)
 
-Deploy some kong proxies which the controllers will test against with:
+## Features
 
-```shell
-$ make deploy.test.proxy
-```
+- **Ingress routing**
+  Use [Ingress][ingress] resources to configure Kong
+- **Enhanced API management using plugins**
+  Use a wide-array of [plugins][kong-hub]
+  to monitor, transform, protect your traffic.
+- **Native gRPC support**
+  Proxy gRPC traffic and gain visibility into it using
+  Kong's plugin.
+- **Health checking and Load-balancing**
+  Load balance requests across your pods and supports active & passive health-checks.
+- **Request/response transformations**
+  Use plugins to
+  modify your requests/responses on the fly.
+- **Authentication**
+  Protect your services using authentication methods
+  of your choice.
+- **Declarative configuration for Kong**
+  Configure all of Kong
+  using CRDs in Kubernetes and manage Kong declaratively.
+
+## Get started
 
-This will deploy the helm chart with the KIC disabled, the Admin API enabled over `ClusterIP`, and 3 proxy replicas.
+You can use
+[Minikube](https://kubernetes.io/docs/setup/minikube/)
+on your local machine or use
+a hosted k8s service like
+[GKE](https://cloud.google.com/kubernetes-engine/).
 
-Now you can run the controller:
+Setting up Kong for Kubernetes is as simple as:
 
 ```shell
-KONG_EXTERNAL_CONTROLLER=true KONG_CONFIGURATION_NAMESPACE=kong-system go run main.go
+# using YAMLs
+$ kubectl apply -f https://bit.ly/k4k8s
+
+# or using Helm
+$ helm repo add kong https://charts.konghq.com
+$ helm repo update
+
+# Helm 3
+$ helm install kong/kong --generate-name --set ingressController.installCRDs=false
 ```
```

Comment:
This doesn't install Railgun.

Reply:
For now should I just remove everything except for the header and basic information?

Reply:
switched to a minimal WIP note instead, but still removing the prototype language as we're past that stage.